apps/sms/views/conversation/js/information.js

```diff
@@ -201,6 +213,15 @@ function createListWithMsgInfo(message) {
   return list;
 }
 
+/**
+ * Will return operator name depending on the iccId.
+ * Will return the empty string in a single SIM scenario.
+ */
+function getOperatorByIccId(index) {
+  var conn = navigator.mozMobileConnections[index];
```

Comment:
> Move the operator query to information view only because we might not need this method in other panels.

Agree

nit: it seems that it's not longer ```getOperatorByIccId``` :) It's more like ```getOperatorByServiceId(serviceId)``` or ```getOperatorBySimIndex(simIndex)``` or ```getOperatorBySimId(simId)``` :)